src/devices/tuya.ts

```diff
@@ -6101,6 +6101,85 @@ const definitions: Definition[] = [
             ],
         },
     },
+	{
+		fingerprint: [{
+			modelID: 'TS0601',
+			manufacturerName: '_TZE204_e9ajs4ft',
+		}],
+		model: 'CTL-R1-TY-Zigbee',
+		vendor: 'TuYa',
+		description: 'Creatrol - 24G radar human presence motion sensor.',
+		fromZigbee: [tuya.fz.datapoints],
+		toZigbee: [tuya.tz.datapoints],
+		exposes: [
+			e.illuminance_lux(),
+			e.presence(),
+			e.numeric('presence_sensitivity', ea.STATE_SET).withValueMin(0).withValueMax(100)
+			.withValueStep(1).withUnit('%').withDescription('presence_sensitivity'),
+	
+			e.numeric('detection_range', ea.STATE_SET).withValueMin(1.5).withValueMax(4.5).withValueStep(
+				0.1).withUnit('m').withDescription('Detection range'),
+	
+			e.numeric('detection_delay', ea.STATE_SET).withValueMin(1).withValueMax(600).withValueStep(1)
+			.withUnit('s').withDescription('presence detection delay'),
+	
+			e.numeric('illumin_max', ea.STATE_SET).withValueMin(0).withValueMax(2000).withValueStep(1)
+			.withUnit('Lux').withDescription(
+				'The threshold of the light sensor will be linked to the "light_switch" parameter'),
+	
+			e.numeric('illumin_min', ea.STATE_SET).withValueMin(0).withValueMax(1000).withValueStep(1)
+			.withUnit('Lux').withDescription(
+				'The threshold of the light sensor will be linked to the "light_switch" parameter'),
+			e.enum('presence_illumin_switch', ea.STATE_SET, ['disable', 'enable']).withDescription(
```

Comment:
Can you give some more context on what this option does?

Reply:
This parameter is a bit complex to explain. According to the manufacturer's instructions, "light_switch" is an internal status of the product. It is determined based on whether the current ambient light falls within the range of "illumin_min" and "illumin_max" and the state of "presence" to indicate whether it should be turned on or off. The manufacturer recommends using "light_switch" as a condition for automation. However, I believe that in the Tuya APP, it can be helpful, but in Home Assistant (Ha), there are more optional conditions available to assist us in implementing automation.